packages/react-cosmos-component-playground/package.json

```diff
@@ -8,6 +8,9 @@
     "lib"
   ],
   "main": "lib/index.js",
+  "scripts": {
+    "cosmos": "NODE_ENV=development cosmos"
```

Comment:
No need for this, you can already run `yarn run start:playground` from the root project dir.